salt/client/__init__.py

```diff
@@ -188,15 +188,15 @@ def __read_master_key(self):
             key_user = key_user.replace('\\', '_')
         keyfile = os.path.join(self.opts['cachedir'],
                                '.{0}_key'.format(key_user))
-        # Make sure all key parent directories are accessible
```

Comment:
It was causing my tests to fail. Easier to fix the code than add a work-around in the test. Yes, it is pretty much unrelated.